doc/Random.xml

```diff
@@ -33,6 +33,9 @@
 			<argument index="2" name="weights" type="PoolIntArray" default="null" />
 			<argument index="3" name="cumulative" type="bool" default="false" />
 			<description>
+				Returns an [Array] of size [code]count[/code] of random elements picked from [code]from[/code] using weights from either [code]weights[/code] or values of [code]from[/code] if [code]from[/code] is a [Dictionary] (and if [code]weights[/codes] is empty).
```

Comment:
```suggestion
				Returns an [Array] of randomly picked elements from a [code]sequence[/code], with the number of elements equal to [code]count[/code]. The elements are picked according to integer [code]weights[/code] or an array of values from the [code]sequence[/code] if it's a [Dictionary] and if [code]weights[/codes] is empty.
```

I'll rename `from` to `sequence` myself (requires updating bindings). But you could do it yourself as well.